File: Leap Year/Leap Year.cpp
#include<iostream>
using namespace std;

int main(){
    int year;
    cout<<"Please Enter Year ==> ";
    cin>>year;

    if((year % 4 == 0) || (year % 100 == 0 && year % 400 == 0)){
        cout<<"Given Year is Leap Year";
    }else{
        cout<<"Given year is not Leap Year";
    }
    return 0;
}

File: Leap Year/Leap Year.py
#!/usr/bin/env python3
# -*- coding: utf-8 -*-
"""
Created on Mon Oct  5 11:58:36 2020

@author: ravi
"""

year = int(input("Please Enter Year ==> "))

if year % 4 == 0 or (year % 100 == 0 and year % 400 == 0):
    print("Leap Year")
else:
    print("Not a Leap Year")